src/main/java/com/ft/methodearticleinternalcomponentsmapper/transformation/InternalComponentsMapper.java

```diff
@@ -154,6 +155,7 @@ public InternalComponents map(EomFile eomFile, String transactionId, Date lastMo
                     .build();
             final Summary summary = extractSummary(xpath, valueDocument, transactionId, uuid.toString());
             final String pushNotificationsCohort = extractPushNotificationsCohort(xpath, attributesDocument);
+            final String pushNotificationsText = extractPushNotificationsText(xpath, attributesDocument);
```

Comment:
It's not from attributes, it's from value, your path to notification text is: /doc/lead... etc.